Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/Modifier.cpp

```diff
@@ -279,7 +285,7 @@ namespace ScriptCanvasEditor
             if (!result.tempFileRemovalError.empty())
             {
                 VE_LOG
-                ("Temporary file not removed for %s: %s"
+                    ( "Temporary file not removed for %s: %s"
```

Comment:
Minor: do you intend to have this extra space?

Reply:
Yes, sir.